nmrfx-bom/pom.xml

```diff
@@ -112,7 +116,71 @@
             <dependency>
                 <groupId>org.nd4j</groupId>
                 <artifactId>nd4j-native-platform</artifactId>
-                <version>${dl4j.version}</version>
+                <version>${nd4j.version}</version>
+                <!-- license: Apache License, Version 2.0 -->
+                <exclusions>
+                    <!-- excluding nd4j-native because it includes all platforms -->
+                    <!-- only the necessary ones should be included -->
+                    <exclusion>
+                        <groupId>org.nd4j</groupId>
+                        <artifactId>nd4j-native</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.nd4j</groupId>
+                <artifactId>nd4j-native</artifactId>
+                <version>${nd4j.version}</version>
+                <classifier>linux-x86_64</classifier>
+                <!-- license: Apache License, Version 2.0 -->
+            </dependency>
+            <dependency>
+                <groupId>org.nd4j</groupId>
+                <artifactId>nd4j-native</artifactId>
+                <version>${nd4j.version}</version>
+                <classifier>macosx-x86_64</classifier>
```

Comment:
There doesn't seem to be a build for macosx-arm64 for nd4j, mkl or arpack-ng. Is that a problem?

Reply:
It's a pain that these projects haven't yet updated to having arm64, as Mac's with them have been available for quite a while now.  I'll test this out.  I've got Macs with and without arm64 now so I can make sure it all runs.

I'm not sure if there will ever be MKL for arm, as MKL is from Intel